content/english/support_services/submit.md

```diff
@@ -10,20 +10,37 @@ menu:
+[SciLifeLab](mailto:datacentre@scilifelab.se) or [NBIS](mailto:support@nbis.se)
+can provide support for where and how to share data in a public database. For further links, see below.
 
-For most biological datatypes, there are international databases that can be considered de facto standards. For data types where no suitable international repository is available, your data can be deposited to a [General data repository](../general_data_repository).
+The European Bioinformatics Institute (EBI) hosts many different international data repositories which should be used if appropriate. See their [Covid-19 Data Portal data submission page](https://www.covid19dataportal.org/submit-data).
+
+For data types where no suitable international repository is available, your data can be deposited to a [General data repository](../general_data_repository).
+
+##### Want support on data management planning?
 
 [SciLifeLab](https://www.scilifelab.se) and [NBIS](http://www.nbis.se/) can support you with Data Management Planning early on in the projects to make data sharing more efficient, both through [personal consultations](https://nbis.se/support/supportform/index.php?form=consultation) and by providing a [customised tool](https://dsw.scilifelab.se/) to create Data Management Plans.
 We can also assist you in identifying relevant repositories and common international standards for describing and publishing your data, as well as guide you through the submission process.
 
+* [Request Data Management consultation from NBIS](https://nbis.se/support/supportform/index.php?form=consultation)
+
+##### Depositing data in a public repository
+
+By depositing data in a public repository, you accept that the data will be published and may be used and redistributed according to specific terms, which you sometimes can set yourself, sometimes are set by the repository. Data deposition is considered important and you may get credit and acknowledgement for sharing data. A publication often results in getting a unique identifier for your dataset, which often is required for a journal to accept a manuscript for publication. Data sharing is often required by funders.
+
+If required, data can often be published with a set moratorium, which means that data will not go public until a date that you set.
+
 _Note that data from human individuals might need special considerations when it comes to publishing._
 
-* [Request Data Management consultation from NBIS](https://nbis.se/support/supportform/index.php?form=consultation)
+##### Storing the data at your university or SNIC
+
+In Sweden, raw measurement data that is generated in a project belongs to the university, while the results are owned by the researcher - the so called teacher's exception. Unless the data needs protection because of contractual obligations or personal privacy, for example in sensitive personal data such as biomedical measurements on humans, data is considered public.
```

Comment:
```suggestion
In Sweden, raw measurement data that is generated in a project belongs to the university, while the results are owned by the researcher - the so called teacher's exception. Unless the data needs protection because of contractual obligations or personal privacy, for example in sensitive personal data such as biomedical measurements of humans, data is considered public.
```